full-moon-derive/src/symbols.rs

```diff
@@ -105,33 +122,16 @@ pub fn parse(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
             }
         }
 
-        fn parse_symbol(code: &str) -> IResult<&str, &str> {
-            let combinator = alt((
+        fn parse_symbol(code: &str) -> IResult<&str, Symbol> {
+            use nom::character::complete::alphanumeric1;
+            use nom::combinator::{not, value};
+            use nom::sequence::{terminated};
```

Comment:
Move these out to the beginning (before `use indexmap::IndexMap`), and structure it like `use nom::{character::complete::alphanumeric1, combinator::{not, value}, sequence::terminated}`. Rustfmt should help with this.

Reply:
I've adjusted these to have the nested use structure, but putting them at the top of the file won't work, as this is embedded in a macro.

Reply:
Good point, my mistake.